### ISSUE TITLE
Help - Kimai installation

### ISSUE THREAD

**Issue by remissao** [kevinpapst/kimai2#1714]
I'm trying to install Kimai on Ubuntu 20.04 Server LTS.

I followed the installation guide and everything went smoothly, until I reached the webserver part (https://www.kimai.org/documentation/webserver-configuration.html#nginx).

Is the installation supposed to work with kimai.local, as written in the guide. When I tried to generate the ssl certificate for it, I got an error saying that the certificate can't be generated for it because .local is not a TLD.

I also tried skipping the SSL and adding the ip address of the server listening on port 4000 but I still couldn't access Kimai. 

Can someone point me in the right direction?

**Comment by infeeeee** [comment]
kimai.local is just a placeholder, you should use your own domain there. if you want to use port based host instead of name based, just remove or comment out the `server_name ...` line, and access your server via it's ip and it's port, something like http://192.168.1.2:4000

(But this is an issue tracker for kimai, not an nginx tutorial, so this question shouldn't be posted here. Read nginx's beginner guide: https://nginx.org/en/docs/beginners_guide.html)

**Comment by remissao** [comment]
@infeeeee 

Thank you for your reply and sorry if this is not the appropriate channel.

So I completed all the steps in the Kimai installation guide on Ubuntu 20.04 Server LTS.

I got the SSL certificate installed with a subdomain. However, when I open the url, I get a "File not found" page.

If I try typing the local IP I get a "404 Page Not Found" from nginx.

How do I know if Kimai is running?

I haven't found any information online on how to do this.

I tried checking the /var/logs inside the Kimai directory but it does not exist.

**Comment by remissao** [comment]
So I fixed the problem.
If anyone is interested here's how I did it.
I ran these commands to check the nginx service:
```
service nginx status
sudo service nginx configtest

```
Everything was ok.

Then I checked the actual location of the server directory of Kimai.
It turns out that the root directory of Kimai2 was located at `~/kimai2/public` and not `/var/www/kimai2/public`. So I changed that in the nginx virtual hosts file.

When I open the subdomain url now it works.
